source/Octopus.Tentacle.Upgrader/Program.cs

```diff
@@ -89,7 +89,7 @@ static void LogStartupParameters(IList<string> args)
 
         static string SelectAppropriateMsi(IList<string> args)
         {
-            var msi = Environment.Is64BitOperatingSystem ? args[2] : args[1];
```

Comment:
This is a bit confusing to me. When there are only 2 arguments, do things a new way. Otherwise do things the old way. I wonder how this could be expressed in a way that makes it clear to the reader why there are two slightly different code paths based on the argument count?